forum/users/models.py

```diff
@@ -1,3 +1,26 @@
+from django.core.validators import EmailValidator
 from django.db import models
+import uuid
 
 # Create your models here.
+class Role(models.Model):
+    role_id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    role_name = models.CharField(max_length=50, null=False, unique=True)
+
+class User(models.Model):
+    user_id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    user_name = models.CharField(max_length=100, null=False, unique=True)
+    first_name = models.CharField(max_length=100, null=False)
+    last_name = models.CharField(max_length=100, null=False)
+    email = models.EmailField(unique=True, null=False, max_length=50, validators=[EmailValidator()])
+    password = models.CharField(max_length=50)
+    user_phone = models.CharField(max_length=15)
```

Comment:
There’s no validation in place. You should consider adding a custom validator 
_Custom validation example_
```
from django.core.validators import RegexValidator

phone_validator = RegexValidator(
    regex=r'^\+?1?\d{9,15}$',
    message="Phone number must be entered in the format: '+999999999'. Up to 15 digits allowed."
)

user_phone = models.CharField(validators=[phone_validator], max_length=15)
```
Alternatively, you can use Django’s `PhoneNumberField` (from the `django-phonenumber-field` package) to validate phone numbers.

Reply:
Used **django-phonenumber-field** validator